custom_components/audiconnect/manifest.json

```diff
@@ -9,5 +9,5 @@
   "issue_tracker": "https://github.com/arjenvrh/audi_connect_ha/issues",
   "loggers": ["audiconnect"],
   "requirements": ["beautifulsoup4"],
-  "version": "1.0.0"
+  "version": "1.0.1"
```

Comment:
The release CI will do this.  This will cause it to skip a number or maybe even not generate a release.

Reply:
It was done automatically on my github :D but removed the change manually